docs_zh_CN/deployment/onnxruntime_custom_ops.md

```diff
@@ -1,3 +1,333 @@
-# Onnxruntime 自定义算子
+# ONNX Runtime自定义算子
 
-欢迎有兴趣的朋友一起翻译 MMCV 文档。如有兴趣，请在 [MMCV issue](https://github.com/open-mmlab/mmcv/issues) 提 issue 确定翻译的文档。
+<!-- TOC -->
+
+- [ONNX Runtime自定义算子](#onnx-runtime自定义算子)
+  - [SoftNMS](#softnms)
+    - [描述](#描述)
+    - [模型参数](#模型参数)
+    - [输入](#输入)
+    - [输出](#输出)
+    - [类型约束](#类型约束)
+  - [RoIAlign](#roialign)
+    - [描述](#描述-1)
+    - [模型参数](#模型参数-1)
+    - [输入](#输入-1)
+    - [输出](#输出-1)
+    - [类型约束](#类型约束-1)
+  - [NMS](#nms)
+    - [描述](#描述-2)
+    - [模型参数](#模型参数-2)
+    - [输入](#输入-2)
+    - [输出](#输出-2)
+    - [类型约束](#类型约束-2)
+  - [grid_sampler](#grid_sampler)
+    - [描述](#描述-3)
+    - [模型参数](#模型参数-3)
+    - [输入](#输入-3)
+    - [输出](#输出-3)
+    - [类型约束](#类型约束-3)
+  - [CornerPool](#cornerpool)
+    - [描述](#描述-4)
+    - [模型参数](#模型参数-4)
+    - [输入](#输入-4)
+    - [输出](#输出-4)
+    - [类型约束](#类型约束-4)
+  - [cummax](#cummax)
+    - [描述](#描述-5)
+    - [模型参数](#模型参数-5)
+    - [输入](#输入-5)
+    - [输出](#输出-5)
+    - [类型约束](#类型约束-5)
+  - [cummin](#cummin)
+    - [描述](#描述-6)
+    - [模型参数](#模型参数-6)
+    - [输入](#输入-6)
+    - [输出](#输出-6)
+    - [类型约束](#类型约束-6)
+  - [MMCVModulatedDeformConv2d](#mmcvmodulateddeformconv2d)
+    - [描述](#描述-7)
+    - [模型参数](#模型参数-7)
+    - [输入](#输入-7)
+    - [输出](#输出-7)
+    - [类型约束](#类型约束-7)
+
+<!-- TOC -->
+
+## SoftNMS
+
+### 描述
+
+根据`scores`计算`boxes`的soft NMS。 请阅读[Soft-NMS -- Improving Object Detection With One Line of Code](https://arxiv.org/abs/1704.04503)了解细节。
+
+### 模型参数
+
+| 类型    | 参数名          | 描述                                                    |
+| ------- | --------------- | ------------------------------------------------------- |
+| `float` | `iou_threshold` | 用来判断候选框覆盖度的阈值，取值范围[0, 1]。默认值为0   |
+| `float` | `sigma`         | 高斯方法的超参数                                        |
+| `float` | `min_score`     | NMS的score阈值                                          |
+| `int`   | `method`        | NMS的计算方式, (0: `naive`, 1: `linear`, 2: `gaussian`) |
+| `int`   | `offset`        | 用来计算候选框的宽高(x2 - x1 + offset)。可选值0或1      |
+
+### 输入
+
+<dl>
+<dt><tt>boxes</tt>: T</dt>
+<dd>输入候选框。形状为(N, 4)的二维张量，N为候选框数量。</dd>
+<dt><tt>scores</tt>: T</dt>
+<dd>输入得分。形状为(N, )的一维张量。</dd>
+</dl>
+
+### 输出
+
+<dl>
+<dt><tt>dets</tt>: T</dt>
```

Comment:
```suggestion
<dt><tt>dets</tt>: tensor(int64)</dt>
```

Reply:
Output type should be float32 since dets[:,4] is score.

Reply:
Got it. Maybe you can modify the English version(https://github.com/open-mmlab/mmcv/edit/master/docs/deployment/onnxruntime_custom_ops.md) which is `tensor(int64)`